pmg/templates/index.html

```diff
@@ -71,9 +72,10 @@ <h6 class="date">{{ meeting.date|pretty_date|nbsp|safe }}</h6>
         <div class="col-sm-6 col-sm-offset-3 text-center">
           <h2>Search the PMG website</h2>
 
-          <form role="search" action="/search">
+          <form class="search-form" role="search" action="/search" name="middle">
             <div class="input-group">
               <input type="text" class="form-control" name="q" placeholder="Search the PMG website">
+              <input name="ga" type="hidden" value="search_middle">
```

Comment:
You can use the same utm tags in forms, since this is a GET form, instead of having to use GA events.

```html
<input name="utm_source" type="hidden" value="homepage">
<input name="utm_medium" type="hidden" value="s-middle">
```